### ISSUE TITLE
electrum_path is not configurable

### ISSUE THREAD

**Issue by chpio** [spesmilo/electrum#1764]
```
$ docker-compose up
Starting electrumclient_electrum-client_1
Attaching to electrumclient_electrum-client_1
electrum-client_1 | + chown electrum-client:electrum-client -R /data/electrum-client
electrum-client_1 | + exec gosu electrum-client:electrum-client electrum --electrum_path=/data/electrum-client
electrum-client_1 | usage: electrum [-h] [-v] [-P] [-w WALLET_PATH] <command> ...
electrum-client_1 | electrum: error: unrecognized arguments: --electrum_path=/data/electrum-client
```

**Comment by ecdsa** [comment]
hi, where does that script come from?

**Comment by ecdsa** [comment]
fixed in 0e61408dfadbacc32edda620cd65143fc87fec68
